index.ts

```diff
@@ -9,4 +9,4 @@ import './view/mode/index.js'
 import './view/code/index.js'
 import './view/checkbox/index.js'
 import './view/settings/index.js'
-import './view/benchmark/index.js'
+import './view/benchmark/index.js'
```

Comment:
Can you enable editorconfig in your text editor (or copy settings)?

Reply:
Enabled it, missed this warning